employee.rb

```diff
@@ -0,0 +1,44 @@
+class Employee
+  attr_accessor :salary, :name
+
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+  end
+
+  def give_raise(amount)
+    @salary = (@salary.to_i + amount).to_s
```

Comment:
to_i could be tricky here.  If someone is making $10,000.50, they could lose a bit of money.